### DISCUSSION TITLE
آپدیت

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [mortezakarimi/gentelella-rtl#24] [General]
سلام
نسخه [Bootstrap](https://getbootstrap.com/) 5 هم آمده است . ولی فعلا [Gentelella](https://github.com/ColorlibHQ/gentelella) نسخه 4 است
لطفا نسخه RTL را آپدیت کنید
تشکر

**Comment by mortezakarimi** [comment]
<div dir="rtl">

سلام
متاسفانه برنامه‌‌ای برای توسعه به ورژن جدید ندارم
ولی خوشحال میشم مشارکت داشته باشیم برای بروزرسانی تم
</div>
